### PR TITLE
update proposal order

### PR DESCRIPTION
**Pull request by Maliaotw**
---
name: Pull request
about: Proposa a pull request
title: "Issue762 update proposal order"
labels: 
assignees: ''

---

# Description
1. 修改tab分頁鏈接, 預設引導審查數DESC排序。
2. 透過dj-registry設置字段reviews.enough, 調整足夠的審查數。

# Steps to Test This Pull Request

# Expected behavior

# More Information

**Additional context**

### CODE REVIEW

File: src/templates/default/reviews/_includes/proposal_table.html [hunk 1]
```diff
@@ -40,7 +40,13 @@
 			{% if proposal.accepted == None %}
 				<td>{{ proposal.review_count }}</td>
 			{% endif %}
-            <td class="proposal-title"><div>{{ proposal.title }}</div></td>
+            <td class="proposal-title">
+							{% if proposal.review_count > reviews_enough %}
+								<div style="color: darkgray">{{ proposal.title }}</div>
+							{% else %}
+								<div>{{ proposal.title }}</div>
+							{% endif %}
```
Comment (yychen) [inline src/templates/default/reviews/_includes/proposal_table.html:48]: I think you can put the {% if %}...{% else %}...{% endif %} clause inside the div, something like
```html
<div{% if ... %} class="grayed-out"{% endif %}>...
```
and maybe add a CSS class for this grayed-out color? Take a look at [src/static/css/pages/_proposals.scss](src/static/css/pages/_proposals.scss)

Comment (Maliaotw) [reply src/templates/default/reviews/_includes/proposal_table.html:48]: Ok，I will modify

**Comment by tai271828** [comment]
In Taiwan we may say "預設" rather than "默認" : )

**Comment by yychen** [comment]
剛才跑起來測一下發現我之前漏掉一個東西，預設排序要是 "review 數少的" 排在前面，所以現在剛好相反唷，再麻煩改一下，感恩!

**Pull request merged**